cmake/platform/freebsd/gbm.cmake

```diff
@@ -0,0 +1,4 @@
+set(PLATFORM_REQUIRED_DEPS OpenGLES EGL GBM LibDRM)
+set(PLATFORM_OPTIONAL_DEPS VAAPI)
+set(APP_RENDER_SYSTEM gles)
+list(APPEND PLATFORM_DEFINES -DMESA_EGL_NO_X11_HEADERS -DPLATFORM_SETTINGS_FILE=gbm.xml)
```

Comment:
maybe consider to just include the gbm file from linux like it is done for wayland
https://github.com/xbmc/xbmc/blob/efe30507783ce1fd438df7d0a7d86459dc176585/cmake/platform/freebsd/wayland.cmake